classes/cargo.bbclass

```diff
@@ -17,6 +17,9 @@ export RUST_BACKTRACE = "1"
 B = "${WORKDIR}/target"
 export CARGO_TARGET_DIR = "${B}"
 
+# set if musl needs to be used
+RUST_TARGET_USE_MUSL ??= ""
```

Comment:
I actually didn't intend for this to be in this PR - this is for unrelated changes and I'll remove this from this PR.